.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,7 +1,48 @@
-<!-- Add Issue Number, if linked -->
-Linked Issue: 
+## Related Issue 
 
-<!-- Add change log below -->
-## Changes
--
--
+- Info about the related issue 
```

Comment:
Could you remove this section entirely - Info about the related issue. Just linking the issue would do for us.